app/src/main/java/com/example/dos_thesis/adapter/LocationRecyclerViewAdapter.java

```diff
@@ -0,0 +1,180 @@
+        setColors(R.color.colorPrimaryDark_neutral, R.color.black, R.color.black, R.color.black,
+          R.color.black, R.color.black,
+          R.color.black, R.color.black, R.color.black);
+        setAlphas(card, .37f, .37f, 100f, .37f,
+          100f,
+          .37f);
+        break;
+    }
+
+    card.emojiImageView.setImageDrawable(emojiForCircle);
+    card.constraintUpperColorSection.setBackgroundColor(upperCardSectionColor);
+    card.backgroundCircleImageView.setImageDrawable(backgroundCircle);
+    card.nameTextView.setTextColor(locationNameColor);
+    card.phoneNumTextView.setTextColor(locationPhoneNumColor);
+    card.hoursTextView.setTextColor(locationHoursColor);
+    card.hoursHeaderTextView.setTextColor(locationHoursHeaderColor);
+    card.distanceNumberTextView.setTextColor(locationDistanceNumColor);
+    card.milesAbbreviationTextView.setTextColor(milesAbbreviationColor);
+    card.addressTextView.setTextColor(locationAddressColor);
+    card.phoneHeaderTextView.setTextColor(locationPhoneHeaderColor);
+  }
+
+  private void setColors(int colorForUpperCard, int colorForName, int colorForAddress,
+                         int colorForHours, int colorForHoursHeader, int colorForPhoneNum,
+                         int colorForPhoneHeader, int colorForDistanceNum, int colorForMilesAbbreviation) {
+    upperCardSectionColor = ResourcesCompat.getColor(context.getResources(), colorForUpperCard, null);
+    locationNameColor = ResourcesCompat.getColor(context.getResources(), colorForName, null);
+    locationAddressColor = ResourcesCompat.getColor(context.getResources(), colorForAddress, null);
+    locationHoursColor = ResourcesCompat.getColor(context.getResources(), colorForHours, null);
+    locationHoursHeaderColor = ResourcesCompat.getColor(context.getResources(), colorForHoursHeader, null);
+    locationPhoneNumColor = ResourcesCompat.getColor(context.getResources(), colorForPhoneNum, null);
+    locationPhoneHeaderColor = ResourcesCompat.getColor(context.getResources(), colorForPhoneHeader, null);
+    locationDistanceNumColor = ResourcesCompat.getColor(context.getResources(), colorForDistanceNum, null);
+    milesAbbreviationColor = ResourcesCompat.getColor(context.getResources(), colorForMilesAbbreviation, null);
+  }
+
+  private void setAlphas(ViewHolder card, float addressAlpha, float hoursHeaderAlpha, float hoursNumAlpha,
+                         float phoneHeaderAlpha, float phoneNumAlpha, float milesAbbreviationAlpha) {
+    card.addressTextView.setAlpha(addressAlpha);
+    card.hoursHeaderTextView.setAlpha(hoursHeaderAlpha);
+    card.hoursTextView.setAlpha(hoursNumAlpha);
+    card.phoneHeaderTextView.setAlpha(phoneHeaderAlpha);
+    card.phoneNumTextView.setAlpha(phoneNumAlpha);
+    card.milesAbbreviationTextView.setAlpha(milesAbbreviationAlpha);
+  }
+
+
+  static class ViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
```

Comment:
Our static analysis engine detects that your pull request includes a change that **class name shadows simple name of superclass**. According to [best practice](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-shouldn-t-shadow-simple-name-of-superclass-nm-same-simple-name-as-superclass), it can be exceptionally confusing, creating many situations where you have to look at import statements to resolve references, and creating many opportunities to accidentally define methods that don't override the methods in their superclasses.